blockchain/processing.py

```diff
@@ -166,6 +173,17 @@ def trigger_handler():
 
         elif config[PHASE] == 5:
             self._add_registration(5, self._execute_phase_5, config)
+            self._add_registration("timestamp", self._execute_timestamping, config)
```

Comment:
The first registration is the actual phase 5 function which retrieves records from previous phases. The second registration is the chron for timestamping which happens every 30 seconds? i believe. I might not have that correct but I believe that is what it is.